connexion/decorators/parameter.py

```diff
@@ -77,14 +83,14 @@ def wrapper(*args, **kwargs):
                 kwargs[body_name] = request_body
 
         # Add query parameters
-        for key, value in flask.request.args.items():
+        for key, value in list(flask.request.args.items()) + list(default_query_params.items()):
```

Comment:
You can create a set from two `dict_items` with `flask.request.args.items() | default_query_params.items()`.
It would also be cleaner and cleaner if you created a variable before, for example, `query_parameters = flask.request.args.items() | default_query_params.items()`